packages/jest-runtime/src/__tests__/should_instrument.test.js

```diff
@@ -29,7 +29,13 @@ describe('should_instrument', () => {
 
     it('when testRegex provided and file is not a test file', () => {
       testShouldInstrument('source_file.js', defaultOptions, {
-        testRegex: '.*\\.(test)\\.(js)$',
+        testRegex: [/.*\.(test)\\.(js)$'/],
+      });
+    });
+
+    it('when more than one testRegex is provided and filename is not a test file', () => {
+      testShouldInstrument('source_file.js', defaultOptions, {
+        testRegex: [/.*\_(test)\.(js)$/, /.*\.(test)\.(js)$/, /never/],
       });
```

Comment:
I see that we only test happy path in this test file. It would be great to add at least two smoke tests (1 for testRegex, 1 for testMatch) when the file should not be instrumented (e.g. add 4th `expected` argument to `testShouldInstrument` with default value of `true`).

Reply:
There are already two `describe` sections in this file, the 2nd -- "should return false" -- tests for scenarios that don't match, the unhappy path. I did add a test there already for `testRegex`. 

I do see that the description of the spec was incorrect, I have fixed that, as well as a couple other existing ones that were also incorrect. Maybe this was throwing you off - either way if there's still some additional coverage you'd like to see let me know.

I have also added a test of error condition to `normalize.test.js`.

Reply:
Oh my, I totally missed second `testShouldInstrument`. Should be renamed to `testShouldntInstrument` 😛